MANIFEST.in

```diff
@@ -1,4 +1,4 @@
 recursive-exclude tests *
 include *.txt
 include *.rst
-include serpextract/search_engines.pickle
\ No newline at end of file
+include serpextract/search_engines.pickle*
```

Comment:
Ahh I see python 2/3. Mind naming them `search_engines.py2.pickle` and `search_engines.py3.pickle`?